settings.gradle.kts

```diff
@@ -1,7 +1,14 @@
+enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
```

Comment:
I am not familiar with this

Reply:
Typesafe accessors allow you to transform:
```
dependencies {
  implementation(project(":antlr-kotlin-target"))
}
```
into:
```
dependencies {
  implementation(projects.antlrKotlinTarget)
}
```

Reply:
oh, great!